assets/stylesheets/_sections.scss

```diff
@@ -110,61 +140,160 @@
 }
 .leaderboard__item {
   font-family: $font-stack--retro;
-  font-size: $type--large;
+  font-size: $type--medium;
   color: $white;
+  @media screen and (min-width: $bp-small+100){
+    font-size: $type--large;
+  }
+}
+
+
+.blink{
+  color:$lemon;
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
```

Comment:
Nice! Might make sense to separate the keyframes into an animations partial? (unless that seems overkill?)

Reply:
yes maam